cst438Assignment2/src/main/resources/templates/city_info.html

```diff
@@ -15,5 +15,23 @@ <h2 th:inline="text">[[${cityInfo.name}]]</h2>
    Population:    <span th:inline="text">[[${cityInfo.population}]]</span> <br />
    Local Weather: <span th:inline="text">[[${cityInfo.temp}]] F</span> <br />
    Local Time:    <span th:inline="text">[[${cityInfo.time}]]</span> <br />
+   
+   <br />
+   <br />
+   <br />
+   
+   <form action="#" th:action="@{/cities/reservation}" method="post">
```

Comment:
This is implemented correctly, I don't think you need action="#", but still functions properly.